src/csm_testing/tests/sat_functional/bootprep/test_bootprep.py

```diff
@@ -653,6 +702,10 @@ def run_bootprep(self, bootprep_file: str, bootprep_opts: str = None,
         try:
             result = subprocess.run(shlex.split(command), cwd=self.temp_dir.name, check=check,
                                     stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        except KeyboardInterrupt:
```

Comment:
I'm curious if you found that this was indeed necessary to ensure proper cleanup, and if so, why only during the execution of `sat bootprep` commands.